packages/web-app-files/src/mixins/fileActions.js

```diff
@@ -55,6 +58,12 @@ export default {
     ...mapGetters('Files', ['highlightedFile', 'currentFolder']),
     ...mapGetters(['capabilities', 'configuration']),
 
+    // TODO: Replace this with the useIsFilesAppActive composable
+    // once we port this mixin to something using the composition api
+    isFilesAppActive() {
```

Comment:
Oh wow, is it really sufficient to provide the function here? Are we using the `fileActions` mixin really in all locations that use any of the specific file action mixins?

Reply:
That's why the composable is still used in BatchActions ... 
We include single actions there and not the fileActions mixin.

... but that's a very good point, this has become hard to reason about. I will add this to all the mixins :(